exchanges/bitfinex/bitfinex.go

```diff
@@ -1231,11 +1230,11 @@ func (e *Exchange) GetAccountBalance(ctx context.Context) ([]Balance, error) {
 // Currency -  example "BTC"
 // WalletFrom - example "exchange"
 // WalletTo -  example "deposit"
-func (e *Exchange) WalletTransfer(ctx context.Context, amount float64, currency, walletFrom, walletTo string) (WalletTransfer, error) {
+func (e *Exchange) WalletTransfer(ctx context.Context, amount float64, ccy, walletFrom, walletTo string) (WalletTransfer, error) {
```

Comment:
🐒 🌴 I'll stop suggesting this now ... 
```suggestion
func (e *Exchange) WalletTransfer(ctx context.Context, amount float64, curr, walletFrom, walletTo string) (WalletTransfer, error) {
```